spec/lib/retrieve_title_spec.rb

```diff
@@ -164,6 +164,12 @@
 
       expect { RetrieveTitle.crawl("https://example.com") }.not_to raise_error
     end
+
+    it "ignores SSRF lookup errors" do
+      subject.stubs(:fetch_title).raises(FinalDestination::SSRFDetector::LookupFailedError)
+
+      expect { RetrieveTitle.crawl("https://example.com") }.not_to raise_error
```

Comment:
```suggestion
      expect(RetrieveTitle.crawl("https://example.com")).to eq(nil)
```

What do you think about asserting for the return value instead so that we have an expectation about what the method should return if an error is encountered?

Reply:
I think your suggestion is better, as it asserts what happens, and not just what not happens. I used this assertion to match the other error handler, but I'll go ahead and change both of them. 🙂 

<img width="844" alt="Screenshot 2022-12-28 at 10 10 04 AM" src="https://user-images.githubusercontent.com/5259935/209746303-b8a3a148-2e70-45e2-8044-e9f60b72f4c6.png">